build.gradle

```diff
@@ -167,14 +167,14 @@ if(hasProperty('mod_core_plugin')) {
     }
 }
 */
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = targetCompatibility = compileJava.sourceCompatibility = compileJava.targetCompatibility = '1.8'
 
 compileJava {
     options.debug = true
     options.debugOptions.debugLevel = 'source,lines,vars'
 }
 
+/*
```

Comment:
While this stuff isn't important for locally building (in fact it shouldn't really do anything unless you've set up the configuration correctly), it is *quite* important for the CI/release build process so just disabling it isn't really a good idea.